scripts/stage-release.sh

```diff
@@ -114,47 +125,29 @@ if [[ "${SKIP_VALIDATION:-false}" != true ]]; then
     mvn -q clean verify
 fi
 
-mvn -q versions:set -DnewVersion="${RELEASE_VERSION}" -DgenerateBackupPoms=false -DprocessAllModules=true
+echo "Versioning Kroxylicious-junit-extension as ${RELEASE_VERSION}"
 
+mvn -q versions:set -DnewVersion="${RELEASE_VERSION}" -DgenerateBackupPoms=false
 echo "Validating things still build"
 mvn -q clean install -Pquick
 
 
 RELEASE_TAG="v${RELEASE_VERSION}"
 
-echo "Committing release to git"
+echo "Committing framework release to git"
 git add '**/pom.xml' 'pom.xml'
-git commit --message "Release version ${RELEASE_TAG}" --signoff
+git commit --message "Release Framework version ${RELEASE_TAG}" --signoff
 
 git tag -f "${RELEASE_TAG}"
 
 git push "${REPOSITORY}" "${RELEASE_TAG}" ${GIT_DRYRUN:-}
 
-echo "Deploying release to maven central"
-mvn deploy -Prelease -DskipTests=true -DreleaseSigningKey="${GPG_KEY}" -DprocessAllModules=true ${MVN_DEPLOY_DRYRUN:-}
-
-echo "Creating binary distribution"
-mvn package -Pdist -DskipTests=true
-
-if ! command -v gh &> /dev/null
-then
-    echo "gh command could not be found. Please create a release by hand https://github.com/kroxylicious/kroxylicious/releases/new"
-else
-  echo "Creating GitHub release with binary distribution"
-  ORIGINAL_GH_DEFAULT_REPO=$(gh repo set-default -v | (grep -v 'no default repository' || true))
-  gh repo set-default $(git remote get-url ${REPOSITORY})
-  # create GitHub release via CLI https://cli.github.com/manual/gh_release_create
-  if [[ "${DRY_RUN:-false}" == true ]]; then
-    gh release create "${RELEASE_TAG}" ./kroxylicious-*/target/kroxylicious-*-bin.* --title "${RELEASE_TAG}" --notes-file "CHANGELOG.md" --draft
-    echo "Draft release created. This must be manually published."
-  else
-    gh release create "${RELEASE_TAG}" ./kroxylicious-*/target/kroxylicious-*-bin.* --title "${RELEASE_TAG}" --notes-file "CHANGELOG.md"
-  fi
-fi
+echo "Deploying release"
+mvn -q deploy -Prelease,dist -DskipTests=true -DreleaseSigningKey="${GPG_KEY}" ${MVN_DEPLOY_DRYRUN}
 
 PREPARE_DEVELOPMENT_BRANCH="prepare-development-${RELEASE_DATE}"
 git checkout -b ${PREPARE_DEVELOPMENT_BRANCH} ${TEMPORARY_RELEASE_BRANCH}
-mvn versions:set -DnextSnapshot=true -DnextSnapshotIndexToIncrement="${SNAPSHOT_INCREMENT_INDEX}" -DgenerateBackupPoms=false -DprocessAllModules=true
```

Comment:
Do you know why we had `processAllModules` or why its safe to remove now? 

I think it will ensure everything gets bumped.

Reply:
It was removed by mistake when I copied stuff over from the ext.  We need it because the BOM doesn't share the same parent.